tests/test_sessions_api.py

```diff
@@ -281,3 +283,91 @@ def test_minute_granularity_range(self, get_project_id: Callable[[], int]) -> No
             "type": "invalid_query",
             "message": "Minute-resolution queries are restricted to a 7-hour time window.",
         }
+
+    def test_delegate_with_sessions_entity_subscription(self) -> None:
```

Comment:
Would you mind putting the subscriptions tests on a separate class. This TestSessionsApi is for testing the query API. Basically let's follow the same pattern as https://github.com/getsentry/snuba/blob/master/tests/test_api.py#L2010